frappe/public/js/frappe/ui/messages.js

```diff
@@ -413,7 +413,13 @@ frappe.show_alert = frappe.toast = function (message, seconds = 7, actions = {})
 	}
 
 	if (!$("#dialog-container").length) {
-		$('<div id="dialog-container"><div id="alert-container"></div></div>').appendTo("body");
+		$(`
+			<div id="dialog-container">
+				<div id="alert-container-top"></div>
+				<div id="alert-container-center"></div>
+				<div id="alert-container"></div>
+			</div>
```

Comment:
I think the ID should always remain the same.
Better add a class “alert-container-top”, “alert-container-center” and “alert-container-bottom”.

Reply:
Three different divs with same ID? Shouldn't IDs be unique ideally?

Reply:
Oh, you’re right. I see, we’re setting three divs, not just one of the three.

This points to a more general question:
Do we really want three message containers that may be arbitrary filled with messages?
I can imagine this to lead to inconsistent UI,
Rather I think we might want to make the placement configurable for _all_ messages.

But in the end it would depend on our usecase.
If there are certain messages that should be on top or centered, then we might do it this way, if that’s the only way it works.
If instead, we want _all_ messages be placed on top or centered based on configuration or on circumstances (screen size or whatever), then we definitely wouldn’t want three divs.

Reply:
> If instead, we want all messages be placed on top or centered based on configuration or on circumstances (screen size or whatever), then we definitely wouldn’t want three divs.

I don't think we would want all the alerts to be on top, rather selected ones. 

> If there are certain messages that should be on top or centered, then we might do it this way, if that’s the only way it works.

We can maybe keep 1 div and use a class to alter the position? I can try that, thoughts?

Reply:
Personally I think that **_IF_** we want to use two or three different alert regions depending on circumstances, then we want them to be in separate divs and not move one and the same div around.

I‘m totally unsure if we do want this though. Not really feeling the UX improvement. Do we maybe want the standard alerts to pop out a bit more, so they‘re not overlooked?